timely/src/worker.rs

```diff
@@ -419,15 +419,18 @@ impl<A: Allocate> Worker<A> {
 
         let mut operator = subscope.into_inner().build(self);
 
+        let (internal_summary, _) = operator.get_internal_summary();
+
```

Comment:
If at all possible, I'd like to keep this next to the `set_external_summary()` call just to be clear that they are paired. I'm happy to have it hoisted as well.